specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-01-01/managedClusters.json

```diff
@@ -5628,6 +5631,32 @@
       "type": "string",
       "description": "server fqdn type for credentials to be returned",
       "x-ms-parameter-location": "method"
+    },
+    "FormatParameter": {
```

Comment:
The name `FormatParameter` might be a little bit broad, I suppose `KubeConfigFormatParameter` might be better but I am not sure if the name is bound to our RP.